### PR TITLE
 Add xrp tests.

### PR DESCRIPTION
**Pull request by matiu**

### CODE REVIEW

File: packages/bitcore-wallet-service/test/integration/server.js [hunk 1]
```diff
@@ -4029,14 +4178,19 @@ describe('Wallet service', function() {
               var txOpts = {
                 outputs: [{
                   toAddress: addressStr,
-                  amount: 1e8,
+                  amount: ts,
                 }],
                 gasPrice: 1,
-                // ToDo
                 feeLevel: level,
+                form: fromAddr,
```
Comment (justinkook) [inline packages/bitcore-wallet-service/test/integration/server.js]: from typo

File: packages/bitcore-wallet-service/test/integration/server.js [hunk 2]
```diff
@@ -4051,9 +4205,10 @@ describe('Wallet service', function() {
               var txOpts = {
                 outputs: [{
                   toAddress: addressStr,
-                  amount: 1e8,
+                  amount: ts,
                 }],
                 feeLevel: 'madeUpLevel',
+                form: fromAddr,
```
Comment (justinkook) [inline packages/bitcore-wallet-service/test/integration/server.js]: from typo

**Comment by justinkook** [comment]
reference https://github.com/bitpay/bitcore/pull/2488

File: packages/bitcore-wallet-service/test/integration/server.js (hunk 1 above)
Comment (matiu) [reply packages/bitcore-wallet-service/test/integration/server.js]: fixed.thanks.

**Review by justinkook — APPROVED** [review]

**Pull request merged**